ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1809,6 +1809,26 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-compact
+- as: gcp-ipi-confidential-computing-p2-f7
```

Comment:
I guess the feature does not affect the cluster's integration testing too much, I guess we can decrease the test frequency  a bit to f14.